core/src/main/java/com/graphhopper/routing/util/spatialrules/SpatialRule.java

```diff
@@ -68,7 +68,7 @@ public double getMaxSpeed(String highwayTag, double _default) {
 
         @Override
         public AccessValue getAccessValue(String highwayTag, TransportationMode transportationMode, AccessValue _default) {
-            return AccessValue.ACCESSIBLE;
+            return _default;
```

Comment:
This is a breaking change, but the behavior before didn't make sense if you want to block something by default and only allow it for certain areas.  If there was no rule for the area, it always returned accessible and not the _default.

Will add this to NOTICES